dockers/docker-config-engine/Dockerfile.j2

```diff
@@ -0,0 +1,21 @@
+FROM docker-base
+
+RUN apt-get update
+
+# Dependencies for sonic-cfggen 
+RUN apt-get install -y python-lxml python-jinja2 python-netaddr python-ipaddr python-yaml 
+
+COPY \
+{% for deb in docker_config_engine_debs.split(' ') -%}
```

Comment:
You can also strip whitespace in templates by hand. If you add a minus sign (-) to the start or end of a block (e.g. a For tag), a comment, or a variable expression, the whitespaces before or after that block will be removed:

it is correct